site.json

```diff
@@ -394,6 +394,188 @@
+            "title": "JavaScript Demo: Expressions - Comparison Operators",
+            "type": "js"
+        },
+        "expressionsConditionalOperators": {
+            "baseTmpl": "tmpl/live-js-tmpl.html",
+            "exampleCode": "live-examples/js-examples/expressions-conditionaloperators.html",
+            "fileName": "expressions-conditionaloperators.html",
+            "title": "JavaScript Demo: Expressions - Conditional Operators",
+            "type": "js"
+        },
+        "expressionsDeleteOperator": {
+            "baseTmpl": "tmpl/live-js-tmpl.html",
+            "exampleCode": "live-examples/js-examples/expressions-deleteoperator.html",
+            "fileName": "expressions-deleteoperator.html",
+            "title": "JavaScript Demo: Expressions - Delete Operator",
+            "type": "js"
+        },
+        "expressionsDestructuringAssignment": {
+            "baseTmpl": "tmpl/live-js-tmpl.html",
+            "exampleCode": "live-examples/js-examples/expressions-destructuringassignment.html",
+            "fileName": "expressions-destructuringassignment.html",
+            "title": "JavaScript Demo: Expressions - Destructuring Assignment",
+            "type": "js"
+        },
+        "expressionsFunctionAsteriskExpression": {
+            "baseTmpl": "tmpl/live-js-tmpl.html",
+            "exampleCode": "live-examples/js-examples/expressions-functionasteriskexpression.html",
+            "fileName": "expressions-functionasteriskexpression.html",
+            "title": "JavaScript Demo: Expressions - Function* Expression",
+            "type": "js"
+        },
+        "expressionsFunctionExpression": {
+            "baseTmpl": "tmpl/live-js-tmpl.html",
+            "exampleCode": "live-examples/js-examples/expressions-functionexpression.html",
+            "fileName": "expressions-functionexpression.html",
+            "title": "JavaScript Demo: Expressions - Function Expression",
+            "type": "js"
+        },
+        "expressionsGroupingOperator": {
+            "baseTmpl": "tmpl/live-js-tmpl.html",
+            "exampleCode": "live-examples/js-examples/expressions-groupingoperator.html",
+            "fileName": "expressions-groupingoperator.html",
+            "title": "JavaScript Demo: Expressions - Grouping Operator",
+            "type": "js"
+        },
+        "expressionsInOperator": {
+            "baseTmpl": "tmpl/live-js-tmpl.html",
+            "exampleCode": "live-examples/js-examples/expressions-inoperator.html",
+            "fileName": "expressions-inoperator.html",
+            "title": "JavaScript Demo: Expressions - In Operator",
+            "type": "js"
+        },
+        "expressionsInstanceof": {
+            "baseTmpl": "tmpl/live-js-tmpl.html",
+            "exampleCode": "live-examples/js-examples/expressions-instanceof.html",
+            "fileName": "expressions-inoperator.html",
+            "title": "JavaScript Demo: Expressions - In Operator",
+            "type": "js"
```

Comment:
in several places here, `in` should be `instanceof`

Reply:
thanks